tests/arp/test_wr_arp.py

```diff
@@ -258,52 +259,54 @@ def testWrArp(self, request, duthost, ptfhost, creds):
         logger.info('Warm-Reboot Control-Plane assist feature')
         sonicadmin_alt_password = duthost.host.options['variable_manager'].\
             _hostvars[duthost.hostname]['sonic_default_passwords']
-        ptf_runner(
-            ptfhost,
-            'ptftests',
-            'wr_arp.ArpTest',
-            qlen=PTFRUNNER_QLEN,
-            platform_dir='ptftests',
-            platform='remote',
-            params={
-                'ferret_ip': ptfIp,
-                'dut_ssh': dutIp,
-                'dut_username': creds['sonicadmin_user'],
-                'dut_password': creds['sonicadmin_password'],
-                "alt_password": sonicadmin_alt_password,
-                'config_file': VXLAN_CONFIG_FILE,
-                'how_long': testDuration,
-                'advance': False,
-            },
-            log_file='/tmp/wr_arp.ArpTest.log',
-            is_python3=True
-        )
+        if not skip_traffic_test:
```

Comment:
Sorry nit: Wouldn't it be alright if we just returned from the test at the top instead of just before `ptf_runner` ? 

```
def testWrArp(self, request, duthost, ptfhost, creds, skip_traffic_test):   # noqa F811
    if not skip_traffic_test:
        return None
```

Reply:
Thanks for you comments, I think it's better to return, and it's better to return just before ptf_runner because need to test some variables